packages/layers/src/events/index.tsx

```diff
@@ -49,7 +49,7 @@ export const EventManager: React.FC<any> = ({ children }) => {
         return {
           top,
           left: headingPos.left,
-          width: pos.width,
+          width: headingPos.width,
```

Comment:
Nice catch! But If we use the headingPos width, the indicator doesn't stretch fully till the rightmost edge. What about `width: pos.width - headingPos.left`?

Reply:
I tried what you suggested yesterday and then I saw the line anomaly like didn't reach rightmost edge. agree with your suggestion. will update the code

<img width="176" alt="Screen Shot 2020-05-07 at 16 21 52" src="https://user-images.githubusercontent.com/11748183/81278351-c0e99000-907f-11ea-9137-0281a6d49649.png">